src/composables/request.js

```diff
@@ -83,13 +83,16 @@ const _request = (container, awaitingResponse) => (config) => {
   const {
     fulfillProblem = undefined,
     problemToAlert = undefined,
+    alert: alertOption = true,
     ...axiosConfig
   } = config;
 
   const { data } = axiosConfig;
   // This limit is set in the nginx config. The alert also mentions this number.
   if (data != null && data instanceof File && data.size > 100000000) {
-    alert.danger(i18n.t('mixin.request.alert.fileSize', { name: data.name }));
+    if (alertOption) {
+      alert.danger(i18n.t('mixin.request.alert.fileSize', { name: data.name }));
```

Comment:
In this case, I think we always want to show the alert. The user will need to take action if their file is too large.